### PR TITLE
use try on beta requests

### PR DESCRIPTION
**Pull request by jdorweiler**
Just in case we don't get a response back.

### CODE REVIEW

File: lib/DDGC/Web/Controller/InstantAnswer.pm [hunk 1]
```diff
@@ -934,7 +934,11 @@ sub asana_req {
     $req->header("x-hub-signature" => $header_data);
     $req->content($json_data);
 
-    my $result = $ua->request($req);
+    my $result;
+    try{
+        $result = $ua->request($req);
```
Comment (jbarrett) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: We already have a LWP::UA instance available in `$c->d->http`, no need to instantiate new ones.

**Comment by jbarrett** [comment]
The response should probably be cached? See lines 49 and 58 [here](https://github.com/duckduckgo/community-platform/blob/e014113/lib/DDGC/Web/Controller/Duckduckhack.pm) for examples:

**Comment by jdorweiler** [comment]
@jbarrett thanks. I'll make a task to add caching and `$c->d->http` later on unless you think it's needed now?

**Comment by jdorweiler** [comment]
Tested it by switching from `beta.duckduckgo.com` to some other server that doesn't have the installed.json file.

**Comment by jbarrett** [comment]
Not urgent.

To be clear, the issue was the JSON parser choking on HTTP content - I think you should be checking `$response->is_success` before doing anything with the content?

We should also make a task to stop using the plain JSON module in favour of [JSON::MaybeXS](https://metacpan.org/pod/JSON::MaybeXS)

**Pull request merged**